public/index.html

```diff
@@ -1,6 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<script>
+		window.__setTheme = () => {
+			let OSTheme = window.localStorage.osTheme;
```

Comment:
Could we pass `window.localStorage.osTheme` straight into `setAttribute`?

Alternatively prefer a `const` over a `let` and name `osTheme` for consitency.

Reply:
Removed OSTheme it was a leftover from testing.